deeppavlov/models/morpho_tagger/morpho_tagger.py

```diff
@@ -162,8 +181,8 @@ def build(self):
     def _build_word_cnn(self, inputs):
         """Builds word-level network
         """
-        inputs = kl.Lambda(kb.one_hot, arguments={"num_classes": self.symbols_number_},
-                           output_shape=lambda x: tuple(x) + (self.symbols_number_,))(inputs)
+        inputs = kl.Lambda(kb.one_hot, arguments={"num_classes": self.symbols_number_()},
+                           output_shape=lambda x: tuple(x) + (self.symbols_number_(),))(inputs)
```

Comment:
Would it be better to just remove `symbols_number_()` and `tags_number_()` methods in favor of direct calls to `len(self.symbols)` and `len(self.tags)` respectively?

Reply:
Yeah, done. This shortens the code